backend/iam/models.py

```
@@ -340,6 +343,11 @@ def _create_user(
         logger.info("user created sucessfully", user=user)
 
         if mailing:
+            template = (
```

Ruff (0.8.2)

Local variable `template` is assigned to but never used

Remove assignment to unused variable `template`

(F841)